ios/BetterRailWidget/Live Activity/ActivityUtils.swift

```diff
@@ -188,29 +188,29 @@ func findClosestStationInRoute(route: Route) -> Int {
 }
 
 @available(iOS 16.1, *)
-func getActivityInitialState(route: Route) throws -> BetterRailActivityAttributes.ContentState {
-   // get the current train by finding the departure time which is closest to the current time
-   var status = ActivityStatus.waitForTrain
-   
-     let nextStationId = findClosestStationInRoute(route: route)
-     let train = getTrainFromStationId(route: route, stationId: nextStationId)!
-     
-     if route.trains[0].orignStation != train.orignStation {
-       // not the first train, possibly an exchange!
-       if train.orignStation == nextStationId {
-         status = .inExchange
-       }
-     }
-      if train.orignStation != nextStationId {
-       status = .inTransit
-     }
-     
-     return BetterRailActivityAttributes.ContentState(
-      delay: 0,
-      nextStationId: nextStationId,
-      status: status
-     )
- }
+func getActivityCurrentState(route: Route, delay: Int? = nil) throws -> BetterRailActivityAttributes.ContentState {
```

Comment:
Why is it defaults to `nil` and not `0`?

Reply:
If delay is nil, I use the delay in the route. We use the delay in the route in the initial activity content, but the provided delay to update in the task (so it will be in sync with the last provided delay from the server)